pomm-project/pomm-bundle/2.0/etc/pomm.yaml

```diff
@@ -0,0 +1,10 @@
+pomm:
+    configuration:
+        my_db1:
+            dsn: '%env(DATABASE_DSN)%'
+#            pomm:default: true
```

Comment:
It is possible to uncomment it since the first database session declared is the default one anyway.